engine/src/main/java/org/terasology/world/generation/BaseFacetedWorldGenerator.java

```diff
@@ -20,6 +20,9 @@
 import org.terasology.world.generator.WorldConfigurator;
 import org.terasology.world.generator.WorldGenerator;
 
+/**
+ * The most common implementation of {@link WorldGenerator} based on the idea of Facets
```

Comment:
"The most common implementation"
The word common seems strange in this context. Common in what sense? Most commonly implemented (as in multiple people implement it, and most people implement this) or most commonly used implementation (implemented by one person, and used by most users) or what? Can you replace it with something else?
`A basic implementation` or something.